packages/react-core/src/components/Dropdown/__tests__/DropdownToggle.test.tsx

```diff
@@ -184,6 +184,25 @@ describe('DropdownToggle', () => {
       expect(asFragment()).toMatchSnapshot();
     });
 
+    test('action split button - renders secondary variant', () => {
+      const { asFragment } = render(
+        <DropdownToggle
+          id="Dropdown Toggle"
+          toggleVariant="secondary"
+          splitButtonItems={[<div key="1">test</div>]}
+          splitButtonVariant="action"
+          parentRef={document.createElement('div')}
+        >
+          Dropdown
+        </DropdownToggle>
+      );
+
+      const dropdownToggle = screen.getByRole('button').parentElement;
+
+      expect(dropdownToggle).toHaveClass('pf-m-secondary');
+      expect(asFragment()).toMatchSnapshot();
```

Comment:
I'm wondering whether we need this snapshot test. On the one hand it keeps things consistent before this component's tests get updated to our new standard. @wise-king-sullyman what do you think?

Reply:
That was also the dilemma I had when writing this. All we would need to do is remove the snapshot test part to meet the new standards

Reply:
I would remove the snapshot, I don't think it's really adding value here.